code/online_challenges/src/codechef/NEWSCH/NEWSCH.cpp

```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+
+#include <cstring>
+
+using namespace std;
```

Comment:
What about your other codechef PR. It seems to also have these changes. The STRWN one.